data/coalition/coalition missions.txt

```diff
@@ -3211,3 +3211,588 @@ mission "Heliarchs Buy Jump Drive Mission"
+			`	"From the introduction and the first few pages, interesting it sounded, but run it past the Heliarch reviewers, we must. So, <first>, long and intensive, this interview will be. As very unique, to meet a human like her is, and to adapt her book to our culture, a long and drawn out process it will be."`
+			choice
+				`	"How long will I have to wait?"`
+				`	"Should I stay here to accompany Anna?`
+					goto "accompany"
+			label "wait time"
+			`	"Wait a few hours, you must, but for the entire process, perhaps half a year, wait Anna must," explains Ikiyachee.`
+			choice
+				`	"A few hours? That's quite boring, isn't it?"`
+					goto "boring"
+				`	"Should I stay here to accompany Anna?"`
+					goto "accompany"
+			label "boring"
+			`	"If boring for you, the interview is, then leave you may, to the lounge."`
+			choice
+				`	"Then I'm going to the lounge. Where is it?"`
+				`	(Accompany Anna during the interview.)`
+					goto "stay in interview"
+			label "directions to lounge"
+			`	Ikiyachee gives you directions to the lounge, and you head there to wait out the interview.`
+				goto "lounge"
+			label "accompany"
+			`	"To stay with her during the interview, we allow you to, but a lounge for you, we have. If too boring, the interview is, then leave you may."`
+			choice
+				`	(Wait in the lounge.)`
+					goto "directions to lounge"
+				`	(Accompany Anna during the interview.)`
+			label "stay in interview"
+			`	Once the first questions are asked, Anna is all too happy to begin discussing the contents of her novel in great detail.`
+			`	Under the watchful eye of the two Heliarchs in the room, Anna explains her anti-Quarg sentiments, the reasoning behind her choice to write the book, and the content of the book. She speaks quickly, but the Heliarchs' numerous questions for clarification and elaboration make the process a long one.`
+			`	"Sold many books before coming here, have you?" one of the agents asks. Anna embarrassedly glances in your direction. "Well, no," she replies. "But I've been writing better and better over the past few years."`
+			`	Priyepichia's representative speaks up. "Page count, we must know. Binding process, aware of you are? You want what type, and cover style?" "There are 300 pages. The hardcover book should have a Jump Drive emblazoned on the front cover.`
+			`	"I don't know what the binding type means; couldn't you just decide that yourself?"`
+			`	As you listen, you learn that due to the Coalition's review process to make the book more suitable, Anna will have to stay for about half a year, during which she will have an apartment to live in.`
+			choice
+				`	"Do I have to stay with Anna here for half a year?"`
+					goto "half a year"
+				`	(Say nothing.)`
+			label "rest of interview"
+			`	As the hours pass, Anna's enthusiastic tone remains strong and constant. The Kimek then takes the book, puts it in a nearby filing cabinet and thanks you and Anna.`
```

Comment:
Just grouped the two of you since it felt odd for them to thank you first for Anna's book.
```suggestion
			`	As the hours pass, Anna's enthusiastic tone remains strong and constant. The Kimek then takes the book, puts it in a nearby filing cabinet and thanks the two of you.`
```